spec/openapi.yml

```diff
@@ -12,13 +12,9 @@ servers:
   - url: http://localhost:3000
     description: Local development
   - url: http://ngl-dual-101.testnet.symboldev.network:3000
-    description: A TESTNET node run by NEM Group
+    description: A TESTNET node
   - url: http://ngl-dual-201.testnet.symboldev.network:3000
-    description: A TESTNET node run by NEM Group
-  - url: http://ngl-api-001.symbolblockchain.io:3000
-    description: A MAINNET node run by NEM Group
-  - url: http://ngl-api-301.symbolblockchain.io:3000
-    description: A MAINNET node run by NEM Group
+    description: A TESTNET node
```

Comment:
these will be replaced soon too, do you want to remove them as well?

Reply:
Sure.